src/main/java/com/github/wovnio/wovnjava/Interceptor.java

```diff
@@ -507,4 +511,65 @@ private String switchLang(String body, Values values, HashMap<String, String> ur
 
         return getStringFromDocument(body, doc);
     }
+
+    private final Pattern extractDomain = Pattern.compile("(?:https?:)?//([^/]+)");
+
+    private void changeUrlToPunyCode(Document doc) {
+        changeUrlToPunyCodeAttr(doc, "link", "href");
+        changeUrlToPunyCodeAttr(doc, "a", "href");
+        changeUrlToPunyCodeAttr(doc, "script", "src");
+        changeUrlToPunyCodeAttr(doc, "img", "src");
+        changeUrlToPunyCodeAttr(doc, "iframe", "src");
+        changeUrlToPunyCodeAttr(doc, "meta", "content");
```

Comment:
might need to add `changeUrlToPunyCodeAttr(doc, "input", "src");`

Reply:
That's right.
I added it.
Thanks.